fenix/app/src/main/java/org/mozilla/fenix/shopping/store/ReviewQualityCheckState.kt

```diff
@@ -95,8 +95,10 @@ sealed interface ReviewQualityCheckState : State {
              * Denotes no analysis is present for the product the user is browsing.
              */
             data class NoAnalysisPresent(
-                val isReanalyzing: Boolean = false,
-            ) : ProductReviewState
+                val progress: Float? = null,
+            ) : ProductReviewState {
+                val isReanalyzing: Boolean = progress != null && progress <= 100f
```

Comment:
I think here we should set it to `< 100f` since 100f would be completed or other end states.

Reply:
Updated to < 100f